minitests/roi_harness/fasm2bit.sh

```diff
@@ -27,7 +27,7 @@ echo "Design .fasm: $fasm_in"
 echo "Harness .bit: $bit_in"
 echo "Out .bit: $bit_out"
 
-${XRAY_DIR}/tools/fasm2frame.py $fasm_in roi_partial.frm
```

Comment:
there is an env var for this now: XRAY_FASM2FRAMES

Reply:
Sure